content/docs/user-guide/networking/multiplayer/split.md

```diff
@@ -0,0 +1,207 @@
+---
+title: Client and Server Launchers with the Multiplayer Gem
+description: Build Open 3D Engine launchers with the Multiplayer Gem specifically targeting Clients, Servers or both.
+linkTitle: Separating Client and Server
+---
+
+The Multiplayer Gem supports builds that contain only client logic, only server logic, or both client and server logic. This allows users to create executables of smaller size by excluding unnecessary logic and dependencies. It also allows hiding potentially sensitive logic unique to one executable from the other. 
```

Comment:
Would it be appropriate to provide an example of "sensitive logic"? "For example developers are encouraged to ensure their publicly released game client doesn't ship with server logic in order deter hacking."

Reply:
Sure, I don't see why not.